examples/vue/src/main.ts

```diff
@@ -1,5 +1,11 @@
 import { createApp } from 'vue';
 import App from './App.vue';
 import router from './router';
+import AmplifyUIVue from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
 
-createApp(App).use(router).mount('#app');
+// Create app and register plugins
+const app = createApp(App);
+app.use(router);
+app.use(AmplifyUIVue);
+app.mount('#app');
```

Comment:
Remove these changes, as a reviewer it makes me question whether the changes impact the build artifacts